docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,196 @@
+---
+title: DateTime and DateTimeOffset support in System.Text.Json
+description: An overview of how DateTime and DateTimeOffset types are supported in the System.Text.Json library.
+ms.technology: dotnet-standard
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+helpviewer_keywords:
+  - "JSON, Serializer, Utf8"
+  - "JSON DateTime, JSON DateTimeOffset"
+  - "DateTime, DateTimeOffset"
+  - "JsonSerializer, Utf8JsonReader, Utf8JsonWriter, JsonElement, JsonDocument"
+  - "JSON Serializer, JSON Reader, JSON Writer"
+  - "Converter, JSON Converter, DateTime Converter"
+  - "ISO, ISO 8601, ISO 8601-1:2019"
+---
+# DateTime and DateTimeOffset support in System.Text.Json
+
+The System.Text.Json library parses and writes <xref:System.DateTime> and <xref:System.DateTimeOffset> values according to the ISO 8601:-2019 extended profile.
+[Converters](https://docs.microsoft.com/dotnet/api/system.text.json.serialization.jsonconverter-1?view=netcore-3.0) provide custom support for serializing and deserializing with <xref:System.Text.Json.JsonSerializer>.
```

Comment:
Similar to serialization/converters, should we mention (and add samples) for what customization could look like for the reader/writer?

Basically, when writing using `Utf8JsonWriter.WriteStringValue(DateTime)`, we use ISO, but the user can custom format their `DateTime` as a string/utf-8 bytes however they like (either using existing `DateTime.ToString(...)` APIs or `Utf8Formatter`), and then pass that in to `Utf8JsonWriter.WriteStringValue(string)` / `Utf8JsonWriter.WriteStringValue(ReadOnlySpan<byte>)`.

Similarly, when reading other formats, `Utf8JsonReader.GetDateTime()` will fail, but `Utf8JsonReader.GetString()` will give you the JSON string which you can then parse yourself as `DateTime` using the more flexibile/forgiving `DateTime.Parse(...)` and `Utf8Parser` APIs.

Reply:
I'll add this in a follow up PR.